packages/ui/src/validators/hooks/useValidatorsList.tsx

```diff
@@ -17,7 +17,7 @@ export const useValidatorsList = () => {
       key,
       isDescending: key !== state.key ? DESCENDING_KEYS.includes(key) : !state.isDescending,
     }),
-    { key: 'default', isDescending: false }
+    { key: 'commission', isDescending: false }
```

Comment:
Sorry I didn't meant it like that just that... just to sort in ascending order once click the user clicks on it
```suggestion
    { key: 'default', isDescending: false }
```